flybywiresim/aircraft
# fix(GSX): Fixed GSX pin not actually disabling NWS

Fixes #7768 

## Summary of Changes


Does pretty much what is on the tin.

Nose wheel steering should now be disabled when the GSX bypass pin is inserted with the associated memo

## Screenshots (if necessary)



## References





## Additional context



Discord username (if different from GitHub):

## Testing instructions


1. Spawn C/D
2. Do normal setup up to push and start
3. Call GSX pushback
4. Observe ECAM 
5. NWS Disconnect message should show up when bypass pin is inserted
6. For completeness sake one should also do  a normal pushback using our integrated pushback


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
